src/routes/api/lockers/find/[address].js

```diff
@@ -0,0 +1,26 @@
+import { getDB } from "$lib/mongo";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+/** @type {import('@sveltejs/kit').RequestHandler} */
+export async function get(request) {
+  const { address } = request.params;
+
+  const db = await getDB();
+  const walletTokens = db.collection("walletTokens");
+  const lockersCollection = db.collection("lockers");
+
+  const wallets = await walletTokens.find({ token: address }).toArray();
+  const lockers = [];
+
+  for (const { address } of wallets) {
+    const isKenshiLocker = await lockersCollection.findOne({ address });
+    if (isKenshiLocker) lockers.push(address);
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/kenshi-token/www/pullRequest?prid=8700592)
```suggestion
    if (isKenshiLocker) {lockers.push(address);}
```